tests/qa_automation/upload_system_log.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
```

Comment:
libraries should be in `../lib/`directory

Reply:
same as above. Big thanks for your review and comments :+1: